drivers/Kconfig

```diff
@@ -21,4 +21,9 @@ rsource "sensor/Kconfig"
 rsource "serial/Kconfig"
 rsource "wifi/Kconfig"
 
+config NRFE
```

Comment:
The order of the commits is wrong. This symbol should be introduced first, then the zephyr module can be modified to use it.

Reply:
Done